src/platform-integration/web/wasm.md

```diff
@@ -1,53 +1,62 @@
 applications for the web.
 
-{{site.alert.warning}} Development of WebAssembly support for Dart and Flutter
-remains ongoing.
-
-You can preview Wasm and WebAssembly garbage collection (WasmGC) in the
-[`master` channel](https://github.com/flutter/flutter/wiki/flutter-build-release-channels#master).
-As Flutter provides these features as previews,
-ongoing development might result in frequent changes.
-Revisit this page for the latest updates.
-
-**_Last updated {{page.last-update}}_**
-
+{{site.alert.warning}}
+  Development of WebAssembly support for Dart and Flutter remains ongoing.
+  
+  You can preview Wasm and WebAssembly garbage collection (WasmGC)
+  on the [`master` channel][].
+  As Flutter provides these features as previews,
+  ongoing development might result in frequent changes.
+  Revisit this page for the latest updates.
+  
+  **_Last updated {{page.last-update}}_**
 {{site.alert.end}}
 
+[`master` channel]: https://github.com/flutter/flutter/wiki/flutter-build-release-channels#master
+
 ### Background
 
-To compile Dart and Flutter to Wasm, you need a browser that supports
-[WasmGC](https://github.com/WebAssembly/gc/tree/main/proposals/gc). The Wasm
-standard plans to add WasmGC to help garbage-collecting languages like Dart
-execute code in an efficient manner.
+To compile Dart and Flutter to Wasm, you need
+a browser that supports [WasmGC][].
+The Wasm standard plans to add WasmGC to help garbage-collected languages
+like Dart execute code in an efficient manner.
 
-Both the [Chromium V8](https://chromestatus.com/feature/6062715726462976) and
-Firefox teams continue to work on WasmGC. To see the current status on the
-WasmGC and other proposals, check out the
-[WebAssembly roadmap](https://webassembly.org/roadmap/).
+[Chromium and V8][] released stable support for WasmGC in Chromium 119,
+and the Firefox team plans to release stable support in Firefox 120.
+To see the current status of WasmGC and other proposals,
+check out the [WebAssembly roadmap][].
+
+[WasmGC]: https://github.com/WebAssembly/gc/tree/main/proposals/gc
+[Chromium and V8]: https://chromestatus.com/feature/6062715726462976
+[WebAssembly roadmap]: https://webassembly.org/roadmap/
 
 ### Try it out
 
-#### Switch to the Flutter `master` channel.
+#### Switch to the Flutter `master` channel and upgrade
+
+Wasm compilation is only available on the latest builds of the `master` channel.
 
-To learn more about Flutter build release channels, check out the
+To learn more about Flutter build release channels and how to switch to
+the `master` channel, check out the
 [Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels).
 
-Wasm compilation is only available on the `master` channel.
+To then ensure you are running the latest version,
+run `flutter upgrade`.
 
-To verify that you set your environment to the `master` channel, run
-`flutter build web --help`.
+To verify that you correctly set your environment to the `master` channel,
```

Comment:
This could be written in a more concise way.


```suggestion
To verify if your Flutter install uses the `master` channel,
```

Verify and correctly are redundant.